src/components/PasswordInput.vue

```diff
@@ -1,16 +1,14 @@
 <template>
   <div>
-    <div class="text-outline bg-white" :class="borderColor">
-      <div class="text-icon-outline text-icon">
+    <div class="block ml-auto mr-auto bg-white py-2 border w-8/12 flex" >
```

Comment:
Try not to put width in PasswordInput component, as this component is used in other vue files too. Setting the width here might affect the style of the component appearing in other vue files. The width can be set in the parent vue, in the component tag